build.gradle

```diff
@@ -1060,11 +1063,17 @@ task preparePravegaImage(type: Copy) {
 task buildPravegaImage(type: DockerBuildTask, dependsOn: preparePravegaImage) {
     baseTag = pravegaBaseTag
     dockerDir = preparePravegaImage.destinationDir.absolutePath
+    if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
```

Comment:
We could use the following approach 
```
if (JavaVersion.isJava11Compatible()) { // returns true if the current java version is >= Java 11.
    dockerFile = "${dockerDir}/Dockerfile-java-11"
} else {
   dockerFIle = "${dockerDir}/Dockerfile"
}
```
or the other options is to have two tasks for java 8 based docker image and one for java 11 based docker image.

Reply:
I  have made the changes.